backend/src/main/java/com/redhat/cloud/notifications/routers/EndpointResource.java

```diff
@@ -658,6 +668,37 @@ public boolean unsubscribeEmail(
         }
     }
 
+    /**
+     * Sends an integration test event via the specified endpoint.
+     * @param uuid the {@link UUID} of the endpoint to test.
+     * @return a "no content" response on success.
+     */
+    @POST
+    @Path("/{uuid}/test")
+    @Parameters({
+        @Parameter(
+                name = "uuid",
+                in = ParameterIn.PATH,
+                description = "The UUID of the endpoint to test",
+                schema = @Schema(type = SchemaType.STRING)
+            )
+    })
+    @Produces(APPLICATION_JSON)
+    @RolesAllowed(ConsoleIdentityProvider.RBAC_READ_INTEGRATIONS_ENDPOINTS)
```

Comment:
Does it make sense for a user with read permission to test an integration that they didn't create and they can't modify? In case the integration does not work, they don't have any way to know who has the permission to fix it.

Reply:
Does it make sense for a user with read permission to test an integration that they didn't create and they can't modify? In case the integration does not work, they don't have any way to know who has the permission to fix it. Based on that, I would change the permission:
```suggestion
    @RolesAllowed(ConsoleIdentityProvider.RBAC_WRITE_INTEGRATIONS_ENDPOINTS)
```

Reply:
It makes a lot of sense! Changed!